sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/TraceContext.java

```diff
@@ -17,7 +15,7 @@ public interface TraceContext {
    *
    * @return the trace context as an OpenTelemetry context.
    */
-  Context asOpenTelemetryContext();
+  io.opentelemetry.context.Context asOpenTelemetryContext();
```

Comment:
any particular reason for not using the import?

Reply:
not warranted after all, reverted